app/components/landing-page/OurStories/interfaces/StoryProps.interface.tsx

```diff
@@ -0,0 +1,8 @@
+export interface StoryProps {
```

Comment:
Please change the file name format into `[component].interface.tsx` and the class name into [Component]Interface